WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -286,7 +286,7 @@ class AppSettingsViewController: UITableViewController {
                 NSUserActivity.deleteAllSavedUserActivities {}
             }
 
-            let notice = Notice(title: NSLocalizedString("Siri Reset Confirmation", comment: "Notice displayed to the user after clearing the Siri activity donations."), feedbackType: .success)
+            let notice = Notice(title: NSLocalizedString("Siri Reset Confirmation", value: "Successfully cleared Siri Shortcut Suggestions", comment: "Notice displayed to the user after clearing the Siri activity donations."), feedbackType: .success)
```

Comment:
This is the official and recommended way to provide English copies that differ from their key (as opposed to having the developers modify the `.strings` file manually).

Obviously once the whole project is done I'll publish a P2 post explaining the new official process, including how to handle those "key≠English copy" cases.